### ISSUE TITLE
The postgres example does not work out of the box.

### ISSUE THREAD

**Issue by aniket-patel-ef** [actions/example-services#24]
Got this error:
ERRO[0003] Unable to interpolate string '${{ job.services.postgres.ports[5432] }}' - [TypeError: Cannot access member 'ports' of undefined] 
[Postgres Service Example/vm-job       ] ⭐  Run node client.js
| (node:32) UnhandledPromiseRejectionWarning: Error: connect ECONNREFUSED 127.0.0.1:5432
|     at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1137:16)
| (node:32) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
| (node:32) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
| /github/workspace/postgres/client.js:14
|     if (err) throw err
|              ^
| 
| Error: Connection terminated unexpectedly

**Comment by rc-keaton** [comment]
Bumping this - same exact error happened when I tried running it

**Comment by Potherca** [comment]
I am not having this issue _at all_. When I fork this repo and push to `main` all example pass :heavy_check_mark: 

Are you seeing this error on GitHub or locally using`act`?

**Comment by rc-keaton** [comment]
> I am not having this issue _at all_. When I fork this repo and push to `main` all example pass 
> 
> Are you seeing this error on GitHub or locally using`act`?

Locally with `act`. I'm using `act version 0.2.17` on Ubuntu 18.04.3

**Comment by Potherca** [comment]
Yeah, I'm seeing it locally too. I guess we should head over to `act` and see [what we can find there](https://github.com/nektos/act/search?q=Unable+interpolate&type=issues).

@aniket-patel-ef Did you encounter this issue locally or on GitHub? If it is locally, I think this issue can be closed.
